### PR TITLE
Fix a few misspellings, aricle mismatches

### PR DESCRIPTION
**Pull request by Blake-Madden**

### CODE REVIEW

File: docs/install/modify-visual-studio.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ Workloads contain the components you need for the programming language or platfo
 
 1. After the modified workloads or components are installed, choose **Launch** from the Visual Studio Installer to open Visual Studio 2022.
 
-You can also use a [configuration file to modify an existing installation to add or remove components](import-export-installation-configurations.md#programmatically-use-a-configuration-file-to-add-components-to-an-existing-installation). Learn more about using [installation configuration files to configure Visual Studio](import-export-installation-configurations.md).
+You can also use a [configuration file to modify an existing installation to add or remove components](import-export-installation-configurations.md#programatically-use-a-configuration-file-to-add-components-to-an-existing-installation). Learn more about using [installation configuration files to configure Visual Studio](import-export-installation-configurations.md).
```
Comment (Court72) [inline docs/install/modify-visual-studio.md:131]:
```suggestion
You can also use a [configuration file to modify an existing installation to add or remove components](import-export-installation-configurations.md#programmatically-use-a-configuration-file-to-add-components-to-an-existing-installation). Learn more about using [installation configuration files to configure Visual Studio](import-export-installation-configurations.md).
```

File: docs/install/update-a-network-installation-of-visual-studio.md [hunk 2]
```diff
@@ -63,12 +63,12 @@ For details on how to control when update notifications are presented to users,
...
 
-## Programatically update the client machines
+## Programmatically update the client machines
 Administrators can programmatically update the client installations of Visual Studio by either issuing the commands to the client-side installer or by invoking a bootstrapper in the network layout.
 
-### Programatically update Visual Studio by using the Visual Studio Installer
+### programmatically update Visual Studio by using the Visual Studio Installer
```
Comment (Court72) [inline docs/install/update-a-network-installation-of-visual-studio.md:69]:
```suggestion
### Programmatically update Visual Studio by using the Visual Studio Installer
```

**Comment by Court72** [comment]
@cwebster-99,

Can you review the proposed change to your article [docs/python/using-python-cookiecutter-templates.md](https://github.com/MicrosoftDocs/visualstudio-docs/pull/10189/files#diff-556a67827b2736a06c686c8e0545fd6e3dc7e6c947dd0a0de818d04d9a6b1554)? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by Court72** [comment]
@black-madden,

Can you fix the following validation warning?

image

Change `#programatically-update-a-client-that-doesnt-have-internet-access` on line 112 in **docs/install/visual-studio-administrator-guide.md** to `#programmatically-update-a-client-that-doesnt-have-internet-access`.

**Comment by Blake-Madden** [comment]
@Court72

> @Blake-Madden,
> 
> Can you fix the following validation warning?
> Change `#programatically-update-a-client-that-doesnt-have-internet-access` on line 112 in **docs/install/visual-studio-administrator-guide.md** to `#programmatically-update-a-client-that-doesnt-have-internet-access`.

Got it.

**Comment by Jak-MS** [comment]
@Blake-Madden - thanks for all these fixes!

reviewed!  and merging as they consist of typo fixes.

**Pull request merged**